--- ../triangle-source-original/makefile	2005-07-27 19:22:01.000000000 -0500
+++ makefile	2024-09-18 14:01:41.631057575 -0500
@@ -18,7 +18,6 @@
 
 # CC should be set to the name of your favorite C compiler.
 
-CC = cc
 
 # CSWITCHES is a list of all switches passed to the C compiler.  I strongly
 #   recommend using the best level of optimization.  I also strongly
@@ -73,7 +72,7 @@
 #
 #   CSWITCHES = -O -DNO_TIMER -DLINUX -I/usr/X11R6/include -L/usr/X11R6/lib
 
-CSWITCHES = -O -DLINUX -I/usr/X11R6/include -L/usr/X11R6/lib
+CSWITCHES = -O -DLINUX
 
 # TRILIBDEFS is a list of definitions used to compile an object code version
 #   of Triangle (triangle.o) to be called by another program.  The file
